src/scaffold/schemas/schemas.ts

```diff
@@ -0,0 +1,76 @@
+export type Attribute =
```

Comment:
these schemas are also subject to change, but just built off of existing staffing-app-api schemas and added fields that would be useful on frontend